src/bicep/modules/SentinelSolution.bicep

```diff
@@ -0,0 +1,20 @@
+param workspaceName  string
+param workspaceId string
+param location string = resourceGroup().location
+param tags object = {}
+
+resource azurerm_log_analytics_solution 'Microsoft.OperationsManagement/solutions@2015-11-01-preview'={
```

Comment:
I think a resource name like `sentinelSolution` makes it clearer what this resource is